api/src/user/user.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Body, Controller, Patch } from '@nestjs/common';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { UserRO } from './ro/user.ro';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+  constructor(private readonly userService: UserService) {}
+
+  @Patch()
```

Comment:
Do we need to update user info in our system, our scope is to save location info on db.